package.json

```diff
@@ -37,7 +37,7 @@
     "xmlbuilder": "^4.1.0"
   },
   "scripts": {
-    "mypublish": "grunt default test && cp dist/vgl* . && npm publish",
+    "mypublish": "grunt default && cp dist/vgl{.min,}.js . && npm publish",
```

Comment:
The issue was I failed to copy dist/vgl.* to . before typing npm publish.
The grunt command `mypublish` didn't work (if it had, I probably wouldn't
have made the mistake).

On Mon, Jul 11, 2016 at 3:10 PM, Aashish Chaudhary <notifications@github.com
> wrote:

> In package.json
> <https://github.com/OpenGeoscience/vgl/pull/99#discussion_r70318043>:
>
> > @@ -37,7 +37,7 @@
> >      "xmlbuilder": "^4.1.0"
> >    },
> >    "scripts": {
> > -    "mypublish": "grunt default test && cp dist/vgl* . && npm publish",
> > +    "mypublish": "grunt default && cp dist/vgl{.min,}.js . && npm publish",
>
> was this the issue?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenGeoscience/vgl/pull/99/files/48af2e2fe262fb073283fb5a0267f71d8e3faa46#r70318043>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIX_RxF6goCBw7YTnk-aXlOv5lKwAPAyks5qUpUegaJpZM4JJrdo>
> .
>



-- 
David Manthey
R&D Engineer
Kitware Inc.
(518) 881-4439